src/pages/AddProperty/AddProp.jsx

```diff
@@ -45,7 +45,8 @@ class AddProp extends React.Component {
         sqFeet: 0,
         sqFeetLotSize: 0,
         yearBuilt: Date.now(),
-        form: new FormData()
+        form: new FormData(), 
+        draft: false
```

Comment:
this should actually be `status` and we will set it to either `publish` or `draft`

Reply:
This was my bad: I didn't update the backend correctly. Make sure you pull the latest down for the backend to see if this works correctly. I did it on my machine and it works but I wanted to save the work for you 😉  here's some docs about event handling in react: https://reactjs.org/docs/handling-events.html#passing-arguments-to-event-handlers